lib/collectionspace/converter/publicart/acquisition.rb

```diff
@@ -61,7 +64,7 @@ def convert
               'xmlns:xsi' => 'http://www.w3.org/2001/XMLSchema-instance'
             ) do
               xml.parent.namespace = nil
-              PublicArtAcquisition.map_commission(xml, attributes)
+              PublicArtAcquisition.map_commission(xml, attributes, redefined_fields)
```

Comment:
Because this is an extension, it's possible to override it. We don't, here, but we include this for consistency.

Reply:
noted